lib/redisdriver.js

```diff
@@ -323,7 +323,8 @@ RedisDriver.prototype._oplogGetOps = function(cName, docName, from, to, callback
   var self = this;
   this.oplog.getOps(cName, docName, from, to, function(err, ops) {
     if (err) return callback(err);
-    if (ops.length && ops[0].v !== from) throw Error('Oplog is returning incorrect ops');
+    if (ops.length && ops[0].v !== from) 
+      return callback('Missing operations');
```

Comment:
It might be easier to just call `self.getOps(...)` instead of replicating the logic here.